aws/resource_aws_ec2_capacity_reservation.go

```diff
@@ -165,13 +166,16 @@ func resourceAwsEc2CapacityReservationRead(d *schema.ResourceData, meta interfac
 	})
 
 	if err != nil {
-		return fmt.Errorf("Error describing EC2 Capacity Reservations: %s", err)
+		if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "InvalidCapacityReservationId.NotFound" {
```

Comment:
Another way to do this would be the `isAWSErr` helper function: 

```suggestion
		if isAWSErr(err, "InvalidCapacityReservationId.NotFound", "") {
```

If you wanted to make this change and remove the associated `awserr` import (and re-run the tests 🙂) that would be super - but this is perfectly functional as it is. The `isAWSErr` is just a slightly neater-looking style convention we're trying to move towards.

Reply:
Hi @ryndaniels, thanks for the review / comment, will make the change quickly